### PR TITLE
[MM-59330] Migrate tooltips of "components/drafts/draft_actions/action.tsx" to WithTooltip

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/27562
Jira https://mattermost.atlassian.net/browse/MM-59330

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/drafts/draft_actions/action.tsx [hunk 1]
```diff
@@ -3,37 +3,25 @@
...
 };
 
 function Action({name, icon, onClick, id, tooltipText}: Props) {
     return (
         <div className='DraftAction'>
-            <OverlayTrigger
-                className='hidden-xs'
-                delayShow={Constants.OVERLAY_TIME_DELAY}
+            <WithTooltip
+                id={`tooltip_${id}`}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/drafts/draft_actions/action.tsx:22]:
```suggestion
                id={`drafts_action_tooltip_${id}`}
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
@Aryakoste can you also please check the failing CI checks

**Comment by M-ZubairAhmed** [comment]
@Aryakoste any update on this one?

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Comment by amyblais** [comment]
@holopin-bot @Aryakoste Mattermost Most Valued Professional (MVP)

**Pull request merged**